client/my-sites/stats/components/line-chart/index.tsx

```diff
@@ -38,57 +33,75 @@ function StatsLineChart( {
 		? formatTimeTick
 		: ( value: number ) => {
 				const date = new Date( value );
-				return new Date( date ).toLocaleTimeString( moment.locale(), {
-					hour: '2-digit',
-					minute: '2-digit',
-					hour12: true,
+				return date.toLocaleDateString( undefined, {
+					month: 'short',
+					day: 'numeric',
 				} );
 		  };
 
 	const formatViews = ( value: number ) => {
 		return value.toFixed( 0 ).toString();
 	};
 
-	const dataSeries = chartData?.[ 0 ].data || [];
+	const isEmpty = ( chartData?.[ 0 ].data || [] ).length === 0;
+
+	const maxViews = useMemo(
+		() =>
+			Math.max(
+				...chartData.map( ( serires ) => Math.max( ...serires.data.map( ( d ) => d.value ) ) )
+			),
+		[ chartData ]
+	);
+
+	// TODO: we should have this in charts lib.
+	const chartDataSorted = useMemo(
```

Comment:
This fixes the issue when the series is not in the order of date asc, which cuases an issue where the tooltip is not showing.

@annacmc we should deal with the sorting from the charts lib side I think

Reply:
yeah, I think that's wise. Adding to my list ✔️